grafana/mattermost-notification-metrics.json

```diff
@@ -0,0 +1,1273 @@
+                      "value": 1
+                    },
+                    {
+                      "color": "orange",
+                      "value": 5
+                    },
+                    {
+                      "color": "red",
+                      "value": 10
+                    }
+                  ]
+                }
+              }
+            ]
+          }
+        ]
+      },
+      "gridPos": {
+        "h": 9,
+        "w": 12,
+        "x": 12,
+        "y": 7
+      },
+      "id": 5,
+      "options": {
+        "legend": {
+          "calcs": [],
+          "displayMode": "list",
+          "placement": "bottom",
+          "showLegend": true
+        },
+        "tooltip": {
+          "maxHeight": 600,
+          "mode": "single",
+          "sort": "none"
+        }
+      },
+      "pluginVersion": "10.3.3",
+      "targets": [
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "expr": "sum(mattermost_notifications_total{type=\"push\"}) OR vector(0)",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "Push Notifications Sent",
+          "range": true,
+          "refId": "Sent"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "expr": "sum(mattermost_notifications_total_ack{type=\"push\"}) OR vector(0)",
+          "hide": false,
+          "instant": false,
+          "legendFormat": "Push Notifications Acknowledged",
+          "range": true,
+          "refId": "Acked"
+        },
+        {
+          "datasource": {
+            "type": "prometheus",
+            "uid": "${DS_PROMETHEUS}"
+          },
+          "editorMode": "code",
+          "expr": "sum(mattermost_notifications_error{type=\"push\"}) OR vector(0)",
+          "hide": true,
+          "instant": false,
+          "legendFormat": "Push Notification Errors",
+          "range": true,
+          "refId": "Error"
+        },
+        {
+          "datasource": {
+            "type": "__expr__",
+            "uid": "${DS_EXPRESSION}"
+          },
+          "expression": "$Sent-$Acked",
+          "hide": false,
+          "refId": "Not Acknowledged",
+          "type": "math"
+        }
+      ],
+      "title": "Push Notification Stats",
+      "transformations": [
+        {
+          "id": "calculateField",
+          "options": {
+            "alias": "",
+            "binary": {
+              "left": "mattermost_notifications_total_ack{instance=\"host.docker.internal:8067\", job=\"mattermost\", type=\"websocket\"}",
+              "operator": "/",
+              "reducer": "sum",
+              "right": "mattermost_notifications_total{instance=\"host.docker.internal:8067\", job=\"mattermost\", type=\"websocket\"}"
```

Comment:
I am still seeing these. Is this intended?

Reply:
Huh, I didn't realize how many places I added these. Will remove the rest.